fix/upp/spc_fields_rtma.txt

```diff
@@ -0,0 +1,347 @@
+PRES:surface:
```

Comment:
Thanks for incorporating changes into `testbed_fields_rtma.txt`. This `spc_fields_rtma.txt` file may be deleted safely now.